distribution/docs/src/main/resources/content/_managing/attributes-added-by-attribute-overrides.adoc

```diff
@@ -0,0 +1,24 @@
+:title: Attributes Added by Attribute Overrides
```

Comment:
In this context, I interpret "adding attributes" to also include modifying attributes, since the purpose of this section is to figure out where attributes on a metacard come from.

And I would like to keep the section naming consistent if possible 😃 .